Try updating actions/setup-python to version 4.

beauremus opened the pull request:
The version matching is broken from 3.6.8, and it's unclear why. My first stab is to update and see if that fixes it.

beauremus committed:
Try updating actions/setup-python to version 4.

The version matching is broken from 3.6.8, and it's unclear why. My first stab is to update and see if that fixes it.

rneswold reviewed (approved):
Using the latest should help but, even with this change, it seems to have a problem build under 3.6.8. I'll approve it to see if it needs to be installed before it can be used.

beauremus committed:
build: :construction_worker: Copy-paste build job to validate functionality.

I want to validate that I understand the problem with build by copy and
pasting the existing build and modifying Python 3.6.8 to match with a
valid version linux. While keeping the existing build without 3.6.8.

If this works, I'd like for a future push to clean this up with
conditional blocks or something of the sort.

beauremus committed:
Fix the syntax

beauremus commented:
The most recent Linux version they support with Python 3.6.8 is `ubuntu-20.04`. My current build is working, but it's inelegant. I'm looking at best practices now.

beauremus committed:
build: :construction_worker: Use the `include` matrix syntax for a special case.

Using the `include` syntax in the matrix definition allows us to run a
special case definition without running all our builds on a second OS
version.

beauremus committed:
build: :construction_worker: Remove the tick to make "os" work.
